### PR TITLE
chore: legacy update Material and MUX

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Chore: legacy update Material and MUX

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
- [x] [Runtime Tests and/or UI Tests](https://platform.uno/docs/articles/contributing/guidelines/creating-tests.html) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal)
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: samples/Directory.Packages.props [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+    <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.1.0" />
+    <PackageVersion Include="SkiaSharp.Skottie" Version="2.88.2" />
+    <PackageVersion Include="SkiaSharp.Views" Version="2.80.2" />
+    <PackageVersion Include="SkiaSharp.Views.Uno" Version="2.88.2" />
+    <PackageVersion Include="SkiaSharp.Views.Uno.WinUI" Version="2.88.2" />
+    <PackageVersion Include="Uno.Core.Extensions.Compatibility" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
+    <PackageVersion Include="Uno.Cupertino" Version="2.7.0-dev.97" />
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="2.7.0-dev.97" />
```
Comment (agneszitte) [inline samples/Directory.Packages.props:27]: Please keep Uno.Cupertino in sync with Uno.Material

File: src/Directory.Packages.props [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.0.0" />
+    <PackageVersion Include="Microsoft.UI.Xaml" Version="2.7.1" />
+    <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.1.0" />
+    <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.755" />
+    <PackageVersion Include="Uno.Core.Extensions.Collections" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
+    <PackageVersion Include="Uno.Core.Extensions.Logging" Version="4.0.1" />
+    <PackageVersion Include="Uno.Cupertino" Version="2.7.0-dev.97" />
+    <PackageVersion Include="Uno.Cupertino.WinUI" Version="2.7.0-dev.97" />
```
Comment (agneszitte) [inline src/Directory.Packages.props:14]: Please keep Uno.Cupertino in sync with Uno.Material also

File: samples/Directory.Packages.props [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
-	<PropertyGroup>
-		<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-	</PropertyGroup>
-	<ItemGroup>
-		<PackageVersion Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Extensions.Logging.Debug" Version="5.0.0" />
-		<PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.7.0-2.final" />
-		<!--
+  <PropertyGroup>
```
Comment (agneszitte) [inline samples/Directory.Packages.props:2]: Please keep tabs instead of space please

**Review by agneszitte — CHANGES_REQUESTED** [review]

File: samples/Directory.Packages.props (hunk 1 above)
Comment (Arieldelossantos) [reply samples/Directory.Packages.props]: done

File: samples/Directory.Packages.props (hunk 3 above)
Comment (Arieldelossantos) [reply samples/Directory.Packages.props]: fixed

**Review by agneszitte — APPROVED** [review]

**Pull request merged**